src/node_url.cc

```diff
@@ -407,29 +407,62 @@ int64_t ParseIPv4Number(const char* start, const char* end) {
   return strtoll(start, nullptr, R);
 }
 
+// https://url.spec.whatwg.org/#ipv4-number-parser
+bool IsIPv4NumberValid(std::string_view input) {
```

Comment:
Your previous function declaration uses `const` qualifier, but this doesn't can you add it to this one too.

Reply:
done!